src/main/java/com/limechain/sync/warpsync/ChainInformation.java

```diff
@@ -0,0 +1,38 @@
+package com.limechain.sync.warpsync;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.math.BigInteger;
+
+@Getter
+@Setter
+public class ChainInformation {
+    public static final BigInteger GRANDPA_VERSION_SUPPORTING_CURRENT_SET_ID = BigInteger.valueOf(3);
```

Comment:
Isn't it more correct `GRANDPA_CURRENT_SET_ID` ?

Reply:
I think GRANDPA_CURRENT_SET_ID would not make it clear that it is a grandpa version not a set id